api/api/bag_geosearch.py

```diff
@@ -0,0 +1,37 @@
+import logging
+
+import requests
+from django.conf import settings
+
+from import_process.process_xls import get_stadsdeel
+
+
+logger = logging.getLogger(__name__)
+
+
+class BagGeoSearchAPI:
+
+    FEATURE_STADSDEEL = "gebieden/stadsdeel"
+
+    def get_api_search_url(self, lat, lon):
+        return settings.BAG_GEO_SEARCH_API_URL.format(
```

Comment:
I think its better if the `BAG_GEO_SEARCH_API_URL` setting just contains the url, not the query params, those should be part of the `BagGeoSearchAPI` class itself